modules/ROOT/nav.adoc

```diff
@@ -325,7 +326,9 @@ include::cli:partial$cbcli/nav.adoc[]
    **** xref:rest-api:rest-establish-credentials.adoc[Establishing Credentials]
    **** xref:rest-api:rest-name-node.adoc[Naming a Node]
    **** xref:rest-api:rest-configure-memory.adoc[Configuring Memory]
-   **** xref:rest-api:rest-set-up-services.adoc[Assigning Services]
+   **** xref:rest-api:rest-get-memory-information.adoc[Getting Memory Information]
+   **** xref:rest-api:rest-set-up-services.adoc[Assigning Services to a New Node]
+   **** xref:rest-api:rest-set-up-services-existing-nodes.adoc[Assigning Services to an Existing Node]
```

Comment:
This comment is for lines 330 and 331.

Could we rename 
330 to  "Assigning Services to a New Single Node Cluster" and
331 to "Assigning Services to an Existing Cluster Node"

There are these ways to assign services using the REST API:

- when joining a cluster -- you are running a command on a new node to join an existing cluster
- when adding a node -- you are running a command on an existing cluster to add a new node
- when you are setting up a single node cluster -- this would be "Assigning Services to a New Single Node Cluster"
- dynamically add/remove non-Data services on existing nodes of a cluster -- this would be "Assigning Services to an Existing Cluster Node"
-